microsoft-365/security/office-365-security/external-email-forwarding.md

```diff
@@ -63,6 +64,16 @@ Remote domain settings and mail flow rules are independent of the settings in ou
 - You allow automatic forwarding in outbound spam filter policies, but you use mail flow rules or remote domain settings to block automatically forwarded email. In this example, the mail flow rules or remote domain settings will block automatically forwarded messages.
 
 This feature independence allows you to (for example) allow automatic forwarding in outbound spam filter policies, but use remote domains to control the external domains that users can forward messages to.
+## How do determine users that are automatically forwarding in your Organization
+
+You can see information about users that are automatically forwarding messages to external recipients in the Auto-forwarded messages report for cloud based accounts. For on-premises users that are automatically forwarding from on-premises through Office 365 you will need to create a Mail flow rule to track these users.  For instructions on how to create a mail flow rule please see [Mail flow rules (transport rules) in Exchange Online](/exchange/security-and-compliance/mail-flow-rules/mail-flow-rules).
```

Comment:
Sorry left out the link to the report - it should be ... in the [Auto-forwarded messages report](mfi-auto-forwarded-messages-report.md)